src/main/java/com/study/gamereviewrest/model/Game.java

```diff
@@ -0,0 +1,33 @@
+package com.study.gamereviewrest.model;
+
+import lombok.Data;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.data.annotation.Id;
+import org.springframework.data.mongodb.core.mapping.Document;
+import org.springframework.data.mongodb.core.mapping.Field;
+import org.springframework.data.mongodb.core.mapping.MongoId;
+
+import java.util.List;
+
+@Data
+@Document
+public class Game {
+    //use id or MongoId
+//    @Autowired
```

Comment:
remove unnecessary comments

Reply:
why is review a part of game POJO,
have you not read DMBS?
game should be a different object with id field
review should be a different object with a gameId and reviewId field
that's how you'll like a game with a review

Reply:
why would somebody add a game and add a set of reviews at the same time